### DISCUSSION TITLE
All types generation from Java plugin

### DISCUSSION THREAD

**Discussion by ch-priya** [dotansimha/graphql-code-generator#6601] [Q&A]
Currently Java plugin is generating classes for only input types and Enums.
For all plugins like Typescript, Kotlin etc, its generating code for all types. I primarily want to generate types in output of operations.

```
type Query {
    getContent(input: GetContentInput!): GetContentOutput
}
```

Eg: Java plugin is generating class for GetContentInput but not for GetContentOutput.
Do we need to add this feature or is there any configuration that I am missing?

**Comment by dotansimha** [comment]
Currently we don't have a plugin to generate that. PRs are always welcome!
